### DISCUSSION TITLE
星際異攻隊3 99分的感想 !? (微雷)

### DISCUSSION THREAD

**Discussion by KozakHou** [1101-GS4719-C/Python-Community-GS4719-C#1379] [General]
禮拜六去新光影城看了星際異攻隊3，看之前先到一樓的燒肉Like飽餐一頓，一邊看著可愛的女朋友一邊往嘴裡塞肉。

看完的整體感想就是 : 好好看喔
好看的地方就都是好看的，不只視覺跟內容。 
當然一些老套路像是火箭死亡邊緣彌留之際準備跟之前的死去的朋友一起上路時，朋友又說還不是時候把火箭送了伙伴身邊。
這個地方是我唯一不解的。

其他的能評分的地方都正中我的紅心，第一次聽到格魯特說別的話壓哈，聽完，結束走出電影院時剛好下雨，我都分不清臉上是雨還是淚了。

**Comment by harrycoconut** [comment]
星際異攻隊三真的戳中我的淚點，尤其是火箭小時後大吼那段（這邊就不劇透了）我的眼眶直接濕了一圈，星爵跟葛摩菈的感情也是非常的感動，這部笑中有淚的星際異攻隊3是我最喜歡的完結篇，最後，我給這部電影9.2分
